tapalcatl_server/server.go

```diff
@@ -41,8 +43,8 @@ type proxyURL struct {
 }
 
 type patternConfig struct {
-	S3           *s3Config   `json:"omitempty"`
-	File         *fileConfig `json:"omitempty"`
+	S3           *s3Config
+	File         *fileConfig
```

Comment:
I would have thought that `omitempty` would only matter on encoding, but it appears to have changed the decode behavior for me too.